src/otx/algo/modules/conv_module.py

```diff
@@ -0,0 +1,357 @@
+                a = self.act_cfg.get("negative_slope", 0.01)
+            else:
+                nonlinearity = "relu"
+                a = 0
+            kaiming_init(self.conv, a=a, nonlinearity=nonlinearity)
+        if self.with_norm:
+            constant_init(self.norm, 1, bias=0)
+
+    def forward(self, x: torch.Tensor, activate: bool = True, norm: bool = True) -> torch.Tensor:
+        """Forward pass of the ConvModule.
+
+        Args:
+            x (torch.Tensor): Input tensor.
+            activate (bool, optional): Whether to apply activation. Defaults to True.
+            norm (bool, optional): Whether to apply normalization. Defaults to True.
+
+        Returns:
+            torch.Tensor: Output tensor.
+        """
+        layer_index = 0
+        while layer_index < len(self.order):
+            layer = self.order[layer_index]
+            if layer == "conv":
+                if self.with_explicit_padding:
+                    x = self.padding_layer(x)
+                # if the next operation is norm and we have a norm layer in
+                # eval mode and we have enabled `efficient_conv_bn_eval` for
+                # the conv operator, then activate the optimized forward and
+                # skip the next norm operator since it has been fused
+                if (
+                    layer_index + 1 < len(self.order)
+                    and self.order[layer_index + 1] == "norm"
+                    and norm
+                    and self.with_norm
+                    and not self.norm.training
+                    and self.efficient_conv_bn_eval_forward is not None
+                ):
+                    self.conv.forward = partial(self.efficient_conv_bn_eval_forward, self.norm, self.conv)
+                    layer_index += 1
+                    x = self.conv(x)
+                    del self.conv.forward
+                else:
+                    x = self.conv(x)
+            elif layer == "norm" and norm and self.with_norm:
+                x = self.norm(x)
+            elif layer == "act" and activate and self.with_activation:
+                x = self.activate(x)
+            layer_index += 1
+        return x
+
+    def turn_on_efficient_conv_bn_eval(self, efficient_conv_bn_eval: bool = True) -> None:
+        """Turn on the efficient convolution batch normalization evaluation.
+
+        Args:
+            efficient_conv_bn_eval (bool, optional): Whether to enable efficient convolution
+                batch normalization evaluation. Defaults to True.
+        """
+        # efficient_conv_bn_eval works for conv + bn
+        # with `track_running_stats` option
+        if efficient_conv_bn_eval and self.norm and isinstance(self.norm, BatchNorm) and self.norm.track_running_stats:
+            self.efficient_conv_bn_eval_forward = efficient_conv_bn_eval_forward
+        else:
+            self.efficient_conv_bn_eval_forward = None
+
+    @staticmethod
+    def create_from_conv_bn(
```

Comment:
I also didn't see where it is used

Reply:
I'll check it out later and remove it if it's unnecessary.